CONTRIBUTING.md

```diff
@@ -22,6 +23,30 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 kubectl scale deployments/inflate --replicas=3
 ```
 
+### Use local environment instead of GitHub Codespaces. 
+> Tested environment : WSL2
+
+1. **Install tools**
+   * make
+   * kubectl
+   * [helm](https://github.com/helm/helm/releases)
+   * [golang](https://go.dev/dl/) > 1.21
+   * [yq](https://github.com/mikefarah/yq/releases) / [jq](https://github.com/jqlang/jq/releases)
+   * [skaffold](https://skaffold.dev/docs/install/#standalone-binary)
+   * [azure-cli](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
```

Comment:
Can we get a simple bash script to install all of these and all on the same version so all developers are on one set of versions? Preferably pinned to one shared version.

Reply:
I added example commands based on what i tested.
I'm not sure if this is what you intended.